
class AlreadyExistError(Exception):
    pass


class UnsupportedError(Exception):
    pass
